src/reconcile/rec_col.c

```diff
@@ -159,6 +167,9 @@ __wt_bulk_insert_var(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk, bool delet
     /* Initialize the time aggregate that's going into the parent page. See note above. */
     WT_TIME_AGGREGATE_UPDATE(session, &r->cur_ptr->ta, &tw);
 
+    ps.row_count = (int64_t)cbulk->rle;
+    WT_PAGE_STAT_UPDATE(&r->cur_ptr->ps, &ps);
```

Comment:
Here the row_count is the delta to add. WT_PAGE_STAT_UPDATE is a confusing name. May be rename it to WT_PAGE_STAT_ADD

Reply:
Yes, I can rename this macro if it makes things easier.